### PR TITLE
amenities section

### PR DESCRIPTION
**Pull request by grow1**
pls check my work :)

### CODE REVIEW

File: src/sass/pages/room/_amenities.scss [hunk 1]
```diff
@@ -0,0 +1,131 @@
+.amenities {
+  background-color: $main-bg-color;
+}
+
+.amenities-items {
+  list-style-type: none;
```
Comment (svoits) [inline src/sass/pages/room/_amenities.scss]: delete list-style-type
add class 'list'

File: src/sass/pages/room/_amenities.scss [hunk 2]
```diff
@@ -0,0 +1,131 @@
+.amenities {
```
Comment (svoits) [inline src/sass/pages/room/_amenities.scss:1]: add
```
padding-top: 32px;
padding-bottom: 28px;
```

File: src/sass/pages/room/_amenities.scss [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+.amen-main-header {
+  font-family: 'Times New Roman', Times, serif;
+  text-transform: uppercase;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 1.15;
+  color: $black;
+  margin-bottom: 16px;
+}
+
```
Comment (svoits) [inline src/sass/pages/room/_amenities.scss:30]: add 
```
.main-description {
margin-bottom: 32px;
}
```

**Review by svoits — CHANGES_REQUESTED** [review]
padding to margin (between items)

File: src/sass/pages/room/_amenities.scss (hunk 1 above)
Comment (grow1) [reply src/sass/pages/room/_amenities.scss]: done

File: src/sass/pages/room/_amenities.scss (hunk 2 above)
Comment (grow1) [reply src/sass/pages/room/_amenities.scss:1]: done

File: src/sass/pages/room/_amenities.scss (hunk 3 above)
Comment (grow1) [reply src/sass/pages/room/_amenities.scss:30]: 1. added mb to main-description
2. changed paddings to margins in items
3. fixed svg 'right arrow' sizes

**Review by svoits — APPROVED** [review]

**Pull request merged**